site/en/guide/basics.ipynb

```diff
@@ -327,7 +327,7 @@
         "id": "ozLLop0cHeYl"
       },
       "source": [
-        "At `x = 1.0`, `y = f(x) = (1**2 + 3 - 5) = -2`.\n",
+        "At `x = 1.0`, `y = f(x) = (1**2 + 2 - 5) = -2`.\n",
```

Comment:
Thanks @Cheril311 

Also, since we're spelling it out, given:

```
def f(x):
  y = x**2 + 2*x - 5
  return y
```

Let's further expand it @MarkDaoust :

```suggestion
        "At `x = 1.0`, `y = f(x) = (1**2 + 2*1 - 5) = -2`.\n",
```